docs/SqlBindingRelatedRepos.md

```diff
@@ -0,0 +1,35 @@
+# Azure SQL Bindings for Azure Functions 
+ 
+## Docs 
+[Azure SQL bindings for Functions | Microsoft Docs ](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-azure-sql?tabs=in-process%2Cextensionv3&pivots=programming-language-csharp)
+  
+## SQL Bindings VSCode Extension 
+[azuredatastudio/extensions/sql-bindings at main · microsoft/azuredatastudio (github.com)](https://github.com/microsoft/azuredatastudio/tree/main/extensions/sql-bindings)
+[sqltoolsservice/src/Microsoft.SqlTools.ServiceLayer/AzureFunctions at main · microsoft/sqltoolsservice (github.com)](https://github.com/microsoft/sqltoolsservice/tree/main/src/Microsoft.SqlTools.ServiceLayer/AzureFunctions)
+ 
+## Templates 
+Currently we have templates for C#, JS, TS, and Python. 
```

Comment:
Probably not necessary to call out the types specifically - it'll be annoying to update later on. Just say something like `Contains templates for the various types of bindings and supported languages, each under a Sql* folder`